EXAMPLES.md

```diff
@@ -444,3 +445,45 @@ public class LogoutModel : PageModel
     }
 }
 ```
+
+## Using AuthenticationApiClient within a Controller
+We can setup the dependency injection for `Auth0AuthenticationApiClient` using the `WithAuth0AuthenticationApiClient` extension method as below.
```

Comment:
I would first focus on what this is, and how it **does not** connect with anything in the ASP.NET SDK. Using this happens outside of the context of the ASP.NET SDK, and this needs to be called out clearly.

I would also emphasis on the use-case here, and why one would need access to the AuthenticationApiClient,